core/src/main/java/cz/xtf/core/bm/BinaryBuild.java

```diff
@@ -238,49 +192,4 @@ private BuildConfig createBcDefinition() {
 
 		return new BuildConfigBuilder().withMetadata(metadata).withSpec(bcBuilder.build()).build();
 	}
-
-	private String getContentHash() {
-		if (!isCached() || contentHash == null) {
-			try {
-				MessageDigest md = MessageDigest.getInstance("SHA-256");
-				DigestOutputStream dos = new DigestOutputStream(NULL_OUTPUT_STREAM, md);
-
-				writeProjectTar(dos);
-
-				// kubernetes annotation value must not be longer than 63 chars
-				contentHash = Hex.encodeHexString(dos.getMessageDigest().digest()).substring(0, 63);
-			} catch (NoSuchAlgorithmException e) {
-				throw new RuntimeException(e);
-			}
-		}
-
-		return contentHash;
-	}
-
-	private void writeProjectTar(OutputStream os) {
```

Comment:
`BinaryBuild` is abstract class inherited in `BinaryDockerBuild`, `BinarySourceBuild` and `MutableBinarySourceBuild`. I don't like that if `writeProjectTar` and  `getContentHash` are removed here in `BinaryBuild` then it duplicated in inhereted classes. I'm not sure what is the reason for change.

Reply:
It has been moved into `BinaryBuildFromSources`.

Reply:
Ok, got it.